.github/workflows/publish.yaml

```diff
@@ -0,0 +1,27 @@
+name: publish Node.js Package to gpr
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+  release:
+    types: [created]
+  workflow_dispatch:
+
+jobs:
+  publish-gpr:
```

Comment:
Let’s make this publish to both npm and gpr.

Reply:
sure, how does publishing to npm work atm?

Reply:
or more specific: where can I find the NPM secret?

Reply:
it seems like it but alas it's not possible 😩

> Secrets cannot be directly referenced in if: conditionals. Instead, consider setting secrets as job-level environment variables, then referencing the environment variables to conditionally run steps in the job.

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-using-secrets